anylabeling/services/auto_labeling/model.py

```diff
@@ -57,7 +57,7 @@ def get_model_abs_path(self, model_path):
                 " while..."
             )
             relative_path = model_path.replace("anylabeling_assets/", "")
-            download_url = self.BASE_DOWNLOAD_URL + relative_path
+            download_url = self.BASE_DOWNLOAD_URL + relative_path.replace("models/yolov5/","").replace("models/segment_anything/","")
```

Comment:
I think replacing like this is not a best practice.
Please change the code as following:
```py
if model_path.startswith("anylabeling_assets/"):
    download_url = ....
elif model_path.lower().startswith(("http://", "https://")):
    download_url = .....
```